source/isaaclab/isaaclab/sim/spawners/from_files/from_files_cfg.py

```diff
@@ -132,9 +132,22 @@ class UrdfFileCfg(FileCfg, converters.UrdfConverterCfg):
     func: Callable = from_files.spawn_from_urdf
 
 
-"""
-Spawning ground plane.
-"""
+@configclass
+class UsdFileWithPhysicsMaterialOnPrimsCfg(UsdFileCfg):
+    """USD file to spawn asset from with physics material.
+
+    It uses the :meth:`spawn_from_usd_with_physics_material` function to spawn the USD file and apply the physics material.
+    """
+
+    func: Callable = from_files.spawn_from_usd_with_physics_material_on_prim
+    compliant_contact_stiffness: float | None = None
+    compliant_contact_damping: float | None = None
+    apply_physics_material_prim_path: str | None = None
```

Comment:
`apply` sounds more of a function name which makes it confusing. My expectation would be that this is a bool variable which says whether to apply or not. But it is a string. Is there a better name we can use here instead that is less ambiguou?